query/src/main/java/io/keen/client/java/AbsoluteTimeframe.java

```diff
@@ -46,16 +46,20 @@ public AbsoluteTimeframe(String start, String end) {
      */
     @Override
     public Map<String, Object> constructTimeframeArgs() {
-        Map timeframe = null;
+        Map<String, Object> timeframe = null;
+
         if (start != null && end != null) {
-            Map absoluteTimeframe = new HashMap<String, Object>();
+            Map<String, Object> absoluteTimeframe = new HashMap<String, Object>(2);
```

Comment:
Yeah oops. We'd want to divide by the default load factor I imagine if we wanted to keep this minor optimization. I'll change it in a follow-up commit.